nms/app/views/organizations/useAxiosTestHelper.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2022 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import useAxios from '../../hooks/useAxios';
+
+export function mockUseAxios(responses: {[url: string]: any}) {
```

Comment:
Flow is unhappy about the any have to tell it to ignore this. And we can say that is should at least be `{ data: any }`